config/jobs/kubernetes-sigs/kind/kind-presubmits.yaml

```diff
@@ -48,6 +63,13 @@ presubmits:
         # we need privileged mode in order to do docker in docker
         securityContext:
           privileged: true
+        resources:
+          limits:
+            cpu: "2"
+            memory: 4Gi
```

Comment:
we have 9G in the other cluster to avoid flakiness

Reply:
```
      resources:
        limits:
          cpu: "4"
          memory: 9000Mi
        requests:
          cpu: "4"
          memory: 9000Mi
```

Reply:
Updated.